apollo-federation/src/error/mod.rs

```diff
@@ -155,7 +155,9 @@ pub enum CompositionError {
 impl CompositionError {
     pub fn code(&self) -> ErrorCode {
         match self {
-            Self::SubgraphError { .. } => todo!(),
+            Self::SubgraphError { error, .. } => {
+                error.errors().first().expect("At least one error").code()
```

Comment:
I'd like to avoid `.expect()` here, even if we have to make up a fake code (like "ERROR_CODE_MISSING") here.